notmuch-addrlookup.c

```diff
@@ -217,11 +218,47 @@ print_contact_info_mutt (gpointer data, gpointer userdata)
   g_print ("%s\t%s\n", info->mail, info->name);
 }
 
+static void
+print_contact_info_aerc (gpointer data, gpointer userdata)
+{
+  GRegex *replace_re = g_regex_new ("\\\"?([^\\\"]+)\\\"?",
+                                  G_REGEX_OPTIMIZE, 0, NULL);
+  ContactInfo *info = (ContactInfo*) data;
+  g_print ("%s\t%s\n", info->mail,
+          g_regex_replace(
+              replace_re,
+              info->name,
+              strlen(info->name),
```

Comment:
You can pass `-1` here to indicate that the string is null-terminated (which is the usual in C strings) and then we don't need to call `strlen()`:

```suggestion
              -1,
```